plugins/woocommerce-admin/client/customer-effort-score-tracks/product-mvp-feedback-modal-container.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { ProductMVPFeedbackModal } from '@woocommerce/customer-effort-score';
+import { recordEvent } from '@woocommerce/tracks';
+import { getAdminLink } from '@woocommerce/settings';
+import { useFormContext } from '@woocommerce/components';
+import { Product } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY } from './data/constants';
+
+export const ProductMVPFeedbackModalContainer: React.FC = () => {
+	const { values } = useFormContext< Product >();
+	const { hideProductMVPFeedbackModal } = useDispatch( STORE_KEY );
+	const { isProductMVPModalVisible } = useSelect( ( select ) => {
+		const { isProductMVPFeedbackModalVisible } = select( STORE_KEY );
+		return {
+			isProductMVPModalVisible: isProductMVPFeedbackModalVisible(),
+		};
+	} );
+
+	const classEditorUrl = values.id
+		? getAdminLink( `post.php?post=${ values.id }&action=edit` )
+		: getAdminLink( 'post-new.php?post_type=product' );
+
+	const recordScore = ( checked: string[], comments: string ) => {
+		recordEvent( 'product_mvp_feedback', {
+			action: 'disable',
+			checked: checked.join( ',' ),
```

Comment:
I was under the impression that tracks handled converting arrays automatically.  Is this required?

Reply:
Nice catch! I fixed it in the commit [a01ffe6](https://github.com/woocommerce/woocommerce/pull/36544/commits/a01ffe6c12b4933cd48c252fd08c102fe5f539e5)